.sbtopts  

```diff
@@ -0,0 +1,2 @@
+-mem 4096 -J-XX:+UseG1
```

Comment:
Is this something to do with the garbage collection? How comes it was added here?